test/mocha.opts

```diff
@@ -2,6 +2,7 @@ test/helpers/*.js
 test
 --compilers js:babel-core/register
 --require ignore-styles
+--require babel-polyfill
```

Comment:
@wyattjoh 

> I would for the moment, unfortunately, recommend removing async/await until we can find a way to tell babel to only apply to the client mocha tests instead of all tests

Where there's a will, there's a way! This commit would only enable transform-async-to-generator when babel bundles code in the client/ dir. What do you think?

https://github.com/gobengo/talk/commit/405dac292c1da4a0b6e1cfce81eac501de62eef0